crypto/x509/x_req.c

```diff
@@ -45,6 +45,29 @@ static int rinf_cb(int operation, ASN1_VALUE **pval, const ASN1_ITEM *it,
     return 1;
 }
 
+static int req_cb(int operation, ASN1_VALUE **pval, const ASN1_ITEM *it,
+                  void *exarg)
+{
+#ifndef OPENSSL_NO_SM2
+    X509_REQ *ret = (X509_REQ *)*pval;
+
```

Comment:
Could ret ever be NULL?

Reply:
Since this is a callback for ASN.1 object handling, I think this should not be NULL when this callback is called.